itou/templates/apply/process_base.html

```diff
@@ -40,14 +40,21 @@ <h1 class="h1-hero-c1">
                         {% if approval_can_be_suspended_by_siae %}
                             <a
                                 href="{% url 'approvals:suspend' approval_id=job_application.approval.id %}?back_url={{ request.get_full_path|urlencode }}"
-                                class="btn btn btn-link">
+                                class="btn btn-link">
                                 Suspendre
                             </a>
+                        {% elif hire_by_other_siae %}
+                            <a href="#" class="btn btn-link disabled">Suspendre</a>
```

Comment:
Le `disabled` n'a pas l'air de changer le visuel pour les `btn-link`, je sais pas si c'est voulu ou non mais peut-être effacer légèrement le texte pour marquer la différence ?
Et utiliser un `<a>` n'est peut-être pas le mieux vu qu'on ne veux pas vraiment de lien au final :thinking:.

Un truc du style :
```suggestion
                            <span class="btn btn-link disabled text-muted">Suspendre</span>
```